src/app/modules/user/services/user.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  baseUrl = `${environment.timeTrackerApiUrl}/users`;
+
+  loadUser(userId): Observable<any> {
```

Comment:
I think can change `Observable<any>`  to `Observable<string>`

Reply:
thank you for your observation.
Actually an observable of type `User` is returned.
I willl change it !
🏁